.github/workflows/pre-commit.yml

```diff
@@ -2,12 +2,15 @@ name: Pre-commit
 on:
   pull_request:
     branches: [ main, master ]
-  push:
-    branches: [ '**' ]
+#  push:
+#    branches: [ '**' ]
 jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
+      - name: Install tfsec for static analysis of Terraform code to spot potential misconfigurations
+        run: |
+          curl -s https://raw.githubusercontent.com/aquasecurity/tfsec/master/scripts/install_linux.sh | bash
```

Comment:
could you use https://github.com/aquasecurity/tfsec-action ?

Reply:
Doesn't work !
/home/runner/.cache/pre-commit/repoy7tvuzt9/hooks/terraform_tfsec.sh: line 67: tfsec: command not found